src/main_loop.rs

```diff
@@ -93,6 +92,8 @@ pub(crate) struct MainLoopState {
     pub(crate) shell: String,
     pub(crate) device_type: DeviceType,
     pub(crate) use_mpris: bool,
```

Comment:
without any feature flags: `warning: field is never read: use_mpris`. not sure what the best action here is, because we definitly want the command line option to be available even if dbus is not enabled. Maybe just mark this with `#[allow(dead_code)]` with a comment explaning why.